docs/how-to-guides/develop-a-tool/add-category-and-tags-for-tool.md

```diff
@@ -0,0 +1,40 @@
+# Add Category and Tags for Tool
+At times, users require their tools to be readily accessible. To facilitate this, we've implemented the `category` and `tags` features. The category aids in compartmentalizing tools into distinct category folders, whereas the tags allow users to locate tools with corresponding tags.  
+Both category and tags assignments are optional. If a tool isn't assigned a category, it will be displayed in the root folder. Similarly, if no tags are assigned, the tags field will remain empty.
+
+## Prerequisites
+- Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.1.0 or a more recent version.
+
+## How to add category and tags for a tool
+Here we use [an existing tool](https://github.com/microsoft/promptflow/tree/main/examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_file_path_input.yaml) as an example. If you wish to create your own tool, kindly refer to the [create and use tool package](create-and-use-tool-package.md#create-custom-tool-package) guide. You can add the _category_ and _tags_ fields in the tool's YAML like this:
+```yaml
+my_tool_package.tools.tool_with_file_path_input.my_tool:
+  function: my_tool
+  inputs:
+    input_file:
+      type:
+      - file_path
+    input_text:
+      type:
+      - string
+  module: my_tool_package.tools.tool_with_file_path_input
+  name: Tool with FilePath Input
+  description: This is a tool to demonstrate the usage of FilePath input
+  type: python
+  # Add a category and tags as shown below.
+  category: test_tool
+  tags:
+    input: FilePath
+    task: text-generation
+```
+
+## Tool with category and tags shown in VS Code extension
+### Tool with category and tags shown in tools tree
+Follow the [steps](create-and-use-tool-package.md#use-your-tool-from-vscode-extension) to use your tool via the VS Code extension. Your tool will be displayed along with its category and tags:  
```

Comment:
Your tool will be displayed along with its category and tags in tool tree, the category will be shown as folder name:

Reply:
Changed, thanks.